layouts/partials/announcement.html

```diff
@@ -15,7 +15,7 @@
     {{- if or (eq .endTime nil ) (gt ( time .endTime ) now ) -}}
       {{- if not $announcementShown -}}
       {{- $announcementShown = true -}}
-  <div lang="en" id="announcement" style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
+  <div lang="en" id="announcement" style="background-color: #3371e3; display: none; color: #fff; {{ .style | safeCSS }}">
```

Comment:
I wouldn't default to _not_ displaying the banner. Wait until there is a cookie detected.

Even better, use the SCSS and handle this based on whether JavaScript is available (check if the `<html>` element has the `no-js` class set)

Reply:
I tried some experimenting with this. It takes a second for the site to render js (maybe we are missing asset optimization a little bit?)

This part of a second causes a flash of the announcement before it's gone; can we somehow fix this?

It's not a major concern as such, The feature works well regardless